config/secrets.json.example

```diff
@@ -102,12 +102,17 @@
       "filename": "logs/analytics.log"
     }
   },
+
   "api_request_timeout": 60,
   "secret": "Replace this string when installing aggie with something else!!",
   "adminParty": false,
   "experiment": false,
   "experimentFile": "test/end-to-end/fixtures/experiment_reports.json",
   "adminPassword": "letmein1",
   "adminEmail": "aggie-admin@example.com",
-  "detectHateSpeech": false
+  "detectHateSpeech": false,
+  "hateSpeechThreshold": {
+      "threshold": 20e-2,
```

Comment:
maybe add a comment there that this value needs to be between 0 and 1

Reply:
Agree this should be better documented, unfortunately JSON doesn't allow comments... the readme would be a good spot for this.

Is 20% thought to be a sensible default? Most server admins will accept it blindly.